cloudsync_onedrive.py

```diff
@@ -724,8 +724,14 @@ def _upload_large(self, drive_path, file_like, conflict):
                 clen = len(data)             # fragment content size
                 cbto = cbfrom + clen - 1     # inclusive content byte range
                 cbrange = "bytes %s-%s/%s" % (cbfrom, cbto, size)
-                headers = {"Content-Length": clen, "Content-Range": cbrange}
-                r = self._direct_api("put", url=upload_url, data=data, headers=headers)
+                try:
+                    headers = {"Content-Length": clen, "Content-Range": cbrange}
+                    r = self._direct_api("put", url=upload_url, data=data, headers=headers)
+                except (CloudDisconnectedError, CloudTemporaryError) as e:
+                    # Should we backoff here? Or have a max number of retries?
```

Comment:
Don't think a backoff is necessary here, if we did have a max retries then the retry count should be per block, not total retries across all blocks of the file